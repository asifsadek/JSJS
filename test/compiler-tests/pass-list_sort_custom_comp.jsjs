val map1 = {
    "foo" : "hello",
    "bar" : "world"
};

val map2 = {
    "foo" : "programming",
    "baz" : "fun"
};

val map3 = {
    "manz" : "playa",
    "foo" : "world"
};

val maplist = [map1, map2, map3];

val key_comp = /\(m1, m2) => get(m1, "foo") < get(m2, "foo");

List.iter(print, List.sort(key_comp, maplist));
