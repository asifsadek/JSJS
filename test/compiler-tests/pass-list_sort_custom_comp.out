Map { "foo": "hello", "bar": "world" }
Map { "foo": "programming", "baz": "fun" }
Map { "manz": "playa", "foo": "world" }
